uniflow-core/src/main/kotlin/io/uniflow/core/flow/DataFlow.kt

```diff
@@ -32,7 +32,7 @@ interface DataFlow {
     fun <T : UIState> actionOn(stateClass: KClass<T>, onAction: ActionFunction<T>): ActionFlow
     fun <T : UIState> actionOn(stateClass: KClass<T>, onAction: ActionFunction<T>, onError: ActionErrorFunction): ActionFlow
     suspend fun onError(error: Exception, currentState: UIState, flow: ActionFlow) {
-        UniFlowLogger.logError("Uncaught error: $error", error)
+        UniFlowLogger.logError("Uncaught error: $error - ${error.stackTrace}", error)
```

Comment:
https://github.com/uniflow-kt/uniflow-kt/pull/47#issuecomment-716499285

Reply:
answered in comment